docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.10
+
+ENV vcf_validator_version=0.9.4
+ENV NXF_VER=22.10.6
+
+WORKDIR /opt
+
+RUN curl -LJo /usr/local/bin/vcf_validator  https://github.com/EBIvariation/vcf-validator/releases/download/v${vcf_validator_version}/vcf_validator_linux \
+    && curl -LJo /usr/local/bin/vcf_assembly_checker  https://github.com/EBIvariation/vcf-validator/releases/download/v${vcf_validator_version}/vcf_assembly_checker_linux \
+    && chmod 755 /usr/local/bin/vcf_assembly_checker /usr/local/bin/vcf_validator
+
+RUN apt update && apt install -y default-jdk
```

Comment:
default-jdk is a moving target. My vote is to pin to a specific JDK and set JAVA_HOME to that... See [here](https://www.ebi.ac.uk/panda/jira/secure/attachment/44781/tomcat_8.0.27_docker.tar.gz) for an example..